### ISSUE TITLE
Allow embedding images from Instagram in addition to Imgur

### ISSUE THREAD

**Issue by DanielKim1** [NatVIII/rva.rip#58]
Is this even possible? If so, it sure would be nice!

**Comment by NatVIII** [comment]
Yeah dawg, that shit would be awesome. I wanna work on getting a Meta account so I can get a working API key to start using instagram again; but considering all the issues with the cache and that being how the current AI instagram stuff works I'm not so sure.

But yeah, in the future, embedding images from Instagram posts would be sexy as hell.

Even farther in the future, or perhaps an alternative future, I want to essentially just create a microservice that watches instagram accounts and every time they post something new the bot then downloads the description and image, AI Bullshits it like bay.lgbt, and then add it to a simple google calendar which can be interpreted by rva.rip.
